docker/Dockerfile.aarch64-linux

```diff
@@ -1,6 +1,7 @@
-FROM ghcr.io/rake-compiler/rake-compiler-dock-image:1.8.0-mri-aarch64-linux
+# Ensure this version matches the rack-compiler-version in Gemfile
+FROM ghcr.io/rake-compiler/rake-compiler-dock-image:1.9.1-mri-aarch64-linux
```

Comment:
Should we template these files?

Reply:
One downside is that dependabot has a limit of PRs it will open, and it can be hard to know if all dockerfiles were updated properly.